### PR TITLE
Stub all ParameterFile functions

### PR DESCRIPTION
**Pull request by GnomedDev**
This allows for safe usage of the first 0x16 bytes of RaceConfig (after the vtable).

### CODE REVIEW

File: payload/game/system/RaceConfig.hh [hunk 1]
```diff
@@ -127,7 +127,9 @@ public:
 private:
     REPLACE static void ConfigurePlayers(Scenario &scenario, u32 screenCount);
 
+    // Was used for ParameterFile, but now unused and free for replacement.
     u8 _0004[0x0020 - 0x0004];
```
Comment (vabold) [inline payload/game/system/RaceConfig.hh:131]: The destructor must be replaced before it is safe to use 0x04 - 0x1c. `DestroyInstance` must be replaced before it is safe to use 0x1c - 0x20.

**Review by vabold — CHANGES_REQUESTED** [review]

File: payload/game/system/RaceConfig.hh (hunk 1 above)
Comment (stblr) [reply payload/game/system/RaceConfig.hh:131]: There is also a vtable access in `RaceConfig::init`.

Comment (vabold) [reply payload/game/system/RaceConfig.hh:131]: The vtable access in `init` is fine because the dereference at 0x0 is valid. We simply stub the function.

**Comment by GnomedDev** [comment]
This doesn't seem worth it.

**Pull request closed without merging**